mozilla/blurts-server
# Add link to marketing email preferences

# References:

Jira: [MNTOR-3851](https://mozilla-hub.atlassian.net/browse/MNTOR-3851)



# Description

Add link to “Manage your Email Preferences”.

# How to test

1. Visit `/user/settings/notifications`
2. Click “Go to Mozilla email settings”
3. You should be prompted to authenticate with FxA and then see  the option to “Manage your Email Preferences”

[MNTOR-3851]: https://mozilla-hub.atlassian.net/browse/MNTOR-3851?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
